internal/pkg/aws/ecr/ecr.go

```diff
@@ -24,22 +24,22 @@ const (
 	batchDeleteLimit  = 100
 )
 
-type ecrClient interface {
+type api interface {
 	DescribeImages(*ecr.DescribeImagesInput) (*ecr.DescribeImagesOutput, error)
 	GetAuthorizationToken(*ecr.GetAuthorizationTokenInput) (*ecr.GetAuthorizationTokenOutput, error)
 	DescribeRepositories(*ecr.DescribeRepositoriesInput) (*ecr.DescribeRepositoriesOutput, error)
 	BatchDeleteImage(*ecr.BatchDeleteImageInput) (*ecr.BatchDeleteImageOutput, error)
 }
 
-// Service wraps an AWS ECR client.
-type Service struct {
-	ecr ecrClient
+// Ecr wraps an AWS ECR client.
+type Ecr struct {
```

Comment:
nit: `ECR` instead of `Ecr`: https://github.com/golang/go/wiki/CodeReviewComments#initialisms